pywr-core/src/parameters/profiles/monthly.rs

```diff
@@ -30,10 +30,20 @@ impl MonthlyProfileParameter {
     }
 }
 
+fn days_in_year_month(datetime: &NaiveDateTime) -> u32 {
+    match datetime.month() {
```

Comment:
> Use the `chrono::Month` enum?

it doesn't currently have this functionality and can't be accessed directly from a `NaiveDatetime`.

One option could be to implement a trait for `NaiveDatetime` that mirrors some of the time-rs api (see - https://github.com/chronotope/chrono/issues/69#issuecomment-1510506428). Though it might need to a newtype so that it can be easily used across multiple modules/crates.

Reply:
The function works as it is. I think leave it as is.